.github/workflows/npm-publish.yml

```diff
@@ -0,0 +1,27 @@
+name: npm-publish
+on:
+  push:
+    branches:
+      - master # Change this to your default branch
+jobs:
+  npm-publish:
+    name: npm-publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@master
+      - name: Set up Node.js
+        uses: actions/setup-node@master
+        with:
+          node-version: 10.0.0
```

Comment:
Not sure how important node-version is ... but this is a very old version.  Could we choose node 17?